### PR TITLE
Fix templates

### PR DESCRIPTION
**Pull request by LanderU**

### CODE REVIEW

File: datasheet_generator/datasheet_templates/actuator/datasheet_template_rotaryservo.xml [hunk 1]
```diff
@@ -2,76 +2,79 @@
...
+  <property name="min_comm_rate" type="double" unit="Hz" description="minimum frequency at which the component can work">
+      <value></value>
+  </property>
+  <property name="max_comm_rate" type="double" unit="Hz" description="maximum frequency at which the component can work">
+      <value></value>
+  </property>
+  <property name="max_size_msgs" type="double" unit="Mb" description="maximum amount of Mbs the component will send">
+    <value></value>
+  </property>
+	<property name="Vage" type="float32" unit="V">
```
Comment (ahcorde) [inline datasheet_generator/datasheet_templates/actuator/datasheet_template_rotaryservo.xml]: Voltage

File: datasheet_generator/datasheet_templates/sensor/datasheet_template_3dcamera.xml [hunk 2]
```diff
@@ -2,70 +2,70 @@
...
+        <value></value>
     </property>
-    <property name="product_id" type="uint32" description="part number">
-        <value/>
+    <property name="product_id" type="string" description="part number">
+        <value></value>
     </property>
-    <property name="min_comm_rate" type="double" unit="Hz" description="minimum frequency at which the component can work">
-        <value/>
+    <property name="min_comm_rate" type="double" unit="hertz" description="minimum frequency at which the component can work">
```
Comment (ahcorde) [inline datasheet_generator/datasheet_templates/sensor/datasheet_template_3dcamera.xml]: why are you returning to the hole name?

**Comment by LanderU** [comment]
Fixed!

File: datasheet_generator/datasheet_templates/actuator/datasheet_template_rotaryservo.xml (hunk 1 above)
Comment (ahcorde) [inline datasheet_generator/datasheet_templates/actuator/datasheet_template_rotaryservo.xml]: Vage?

Comment (LanderU) [reply datasheet_generator/datasheet_templates/actuator/datasheet_template_rotaryservo.xml]: Fixed!

**Pull request merged**